package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "wikidata-mismatch-finder",
+    "name": "html",
```

Comment:
This doesn’t look nice – we should probably add the `name` to `package.json` instead. (Looks like @chukarave added it to `package-lock.json` in #489?)

Reply:
⇒ #528

Reply:
This is automatic. Yeah, i guess it will be better in package.json

Reply:
I merged that PR, can you try rebasing / repeating this PR and see what happens to `package-lock.json` now?